sources/core/Xenko.Core/Native/NativeLibrary.cs

```diff
@@ -108,6 +131,17 @@ public static void UnLoadAll()
 #endif
         }
 
+        static void NormalizeLibName(ref string libName)
+        {
+#if XENKO_PLATFORM_WINDOWS_DESKTOP
```

Comment:
Let's exclude the whole method from non-Windows build. That will remove one entry from the type metadata.